src/components/Input/useInput.ts

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+
+import { InputTypeEnum } from '../../../typings/commonTypes';
+import { UseInputProps } from './types';
```

Comment:
🧐
Если это не автоматическая сортировка плагином, то давай отсортируем по возрастанию заборчика)
```suggestion
import { UseInputProps } from './types';
import { InputTypeEnum } from '../../../typings/commonTypes';
```

Reply:
Теперь у меня UseInputProps и InputTypeEnum импортируются из одного места